src/derived-states.js

```diff
@@ -24,13 +23,6 @@ DerivedState.TransitionsSparse = {
     40: {1: 2, 34: 4},
     42: {1: 2, 43: 6},
     43: {1: 2},
-    44: {1: 1, 44: 1},
-    45: {44: 1, 53: 1},
-    46: {48: 1},
-    48: {48: 1, 49: 1},
-    49: {48: 1, 50: 1},
-    50: {1: 1},
-    53: {46: 1}
 };
```

Comment:
`TransitionsSparse` is not used in practice, but apparently designed only for generating the `Transitions` table.
Consider commenting it out?